kalite/distributed/static/js/distributed/video/hbtemplates/video-player.handlebars

```diff
@@ -1,9 +1,13 @@
 <div id="video-player"{{#unless content_urls }} class="client-online-only"{{/unless}}>
     <video class="video-player video-js vjs-default-skin vjs-fullscreen" id="{{ random_id }}">
         <source src="{{ content_urls.stream }}" type="{{ content_urls.stream_type }}" />
-        {{#each subtitle_urls }}
-        <track kind="captions" src="{{ this.url }}" srclang="{{ code }}" label="{{ name }}" {{#ifcond code "==" ../selected_language }}{{#ifcond ../code "!=" "en" }}default="True"{{/ifcond}}{{/ifcond}}/>
-        {{/each}}
+        {{#current_language_is "en"}}
+            {{#ifcond translated_youtube_lang "==" "en"}}
+                {{#each subtitle_urls}}
+                    <track kind="captions" src="{{ this.url }}" srclang="{{ this.code }}" label="{{ this.name }}" {{#ifcond this.code "==" "en"}}default{{/ifcond}}/>
```

Comment:
I think it has to say `{{#current_language_is this.code}} default{{/current_language_is}}` in order to switch on the subtitles by default in case they match the currently set language and the video is in English (which is handled by the outer condition).